shiftcalendar/__init__.py

```diff
@@ -101,7 +102,22 @@ def fill_CalendarEntry_from_Legacy():
         CalendarEntry.insert_many(new_entries).execute()
 
 
+def fill_MoonBreak_from_Legacy():
+
+    old_entries = sorted(
+        list(LegacyCalendarEntry.select()),
```

Comment:
ah now, I cannot use order by can I, have to order by columns, `y`, `m` and then `d`... doable but ugly. I rather use sort on date.